gps2val/requirements.txt

```diff
@@ -1,2 +1,2 @@
 kuksa-viss-client
```

Comment:
This should be kuksa-client now

Reply:
Is it already available under kuksa-client?

Reply:
Yes! https://pypi.org/project/kuksa-client/